projects/abtesting/src/app/core/auth/auth.data.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from '../../../environments/environment';
+
+@Injectable()
+export class AuthDataService {
+  constructor(private http: HttpClient) {}
+
+  createUser(userInfo: any) {
+    const url = environment.api.users;
+    const { token: idToken } = userInfo;
+    delete userInfo['token'];
+    const headers = {
```

Comment:
Instead of adding header in each request you can follow this pattern https://stackoverflow.com/questions/34464108/angular-set-headers-for-every-request

Reply:
Good way, will update it